src/PostgREST/Error.hs

```diff
@@ -271,7 +271,9 @@ pgErrorStatus authed (SQL.SessionError (SQL.QueryError _ _ (SQL.ResultError rErr
         "P0001"   -> HTTP.status400 -- default code for "raise"
         'P':'0':_ -> HTTP.status500 -- PL/pgSQL Error
         'X':'X':_ -> HTTP.status500 -- internal Error
-        "42883"   -> HTTP.status404 -- undefined function
+        "42883"-> if BS.unpack m == "function xmlagg(text) does not exist"
```

Comment:
Hm, this will not catch other parameter types. How about using [isPrefixOf](https://hackage.haskell.org/package/bytestring-0.11.3.0/docs/Data-ByteString-Char8.html#v:isPrefixOf) and catching `function xmlagg(`?

Reply:
OK, done.